pkg/kubeclient/client.go

```diff
@@ -0,0 +1,89 @@
+package kubeclient
```

Comment:
Have you considered any other name, such as `client`?

Reply:
Done.